integration-test/1423-kinds-for-maki-icon-library-support.py

```diff
@@ -2669,3 +2668,530 @@ def test_waterway_amenity_fuel_node(self):
                 'id': 270602036,
                 'kind': u'waterway_fuel',
             })
+
+    def test_bureau_de_change_node(self):
+        import dsl
+
+        z, x, y = (16, 19294, 24641)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/node/2034011032
+            dsl.point(2034011032, (-74.013403, 40.706036), {
+                'amenity': u'bureau_de_change',
+                'name': u'Travelex',
+                'source': u'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'pois', {
+                'id': 2034011032,
+                'kind': u'bureau_de_change',
+            })
+
+    def test_bureau_de_change_way(self):
+        import dsl
+
+        z, x, y = (16, 18993, 23478)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/292975938
+            dsl.way(292975938, dsl.tile_box(z, x, y), {
+                'addr:housenumber': u'1724',
+                'addr:street': u'Bank Street',
+                'amenity': u'bureau_de_change',
+                'building': u'commercial',
+                'name': u'Money Mart',
+                'office': u'financial',
+                'shop': u'money_lender',
```

Comment:
I don't think we have `money_lender` but if we did it'd probably win over `bureau_de_change` during filtering. But ignoring that for now.